darkside-tests/src/chain_generic_tests.rs

```diff
@@ -42,83 +36,94 @@ proptest! {
 ///   - transparent sends do not work
 ///   - txids are regenerated randomly. zingo can optionally accept_server_txid
 /// these tests cannot portray the full range of network weather.
-impl ConductChain for DarksideEnvironment {
-    async fn setup() -> Self {
-        DarksideEnvironment::new(None).await
-    }
+pub(crate) mod impl_conduct_chain_for_darkside_environment {
```

Comment:
I think symmetry is important, can we name this `darkside_environment` to match `libtonode_environment` which serves the same purpose of including the impl for conduct chain? this mod name seems too long and specific

Reply:
well, this mod doesnt include the actual definition of the struct, so it is importantly different from the other